src/templates/drawer.jsx

```diff
@@ -277,7 +280,7 @@ class Drawer extends React.Component {
         } = this.props;
         const animationEvent = domUtils.cssTransitionType(this.drawerContainerRef);
         const boxShadowPositionX = this.isPositionX('right') ? '-' : '';
-        const drawerLength = document.querySelectorAll('.ui.drawer').length;
+        const drawerLength = document.querySelectorAll('.ui.drawer-is_modal').length;
```

Comment:
Should we consider renaming `drawerLength` to something like `numberOfModalDrawers`?
Yeah, it was calculated from the `.length` property of an array, but what's important to understanding the logic is that it represents a _count_ of--or _quantity_ of--things that we're interested in, right?  So in my opinion, this rename would improve readability of the code slightly.

Reply:
Rename looks good; resolving.